apps/editors/tests/test_helpers.py

```diff
@@ -921,7 +916,8 @@ def test_nominated_review_time_set(self):
         for status in REVIEW_ADDON_STATUSES:
             for process in ['process_sandbox', 'process_preliminary',
                             'process_public']:
-                if (status == amo.STATUS_UNREVIEWED and
+                if ((status == amo.STATUS_UNREVIEWED or
+                        status == amo.STATUS_LITE) and
```

Comment:
I would've done `status in (amo.STATUS_UNREVIEWED, amo.STATUS_LITE)`, but this works too.